.github/workflows/consumer_contract_tests.yml

```diff
@@ -109,7 +109,7 @@ jobs:
                           -v jar-cache:/root/.ivy2 \
                           -w /working \
                           sbtscala/scala-sbt:openjdk-17.0.2_1.7.2_2.13.10 \
-                          sbt "project pact4s" clean test
+                          sbt "project pact4s" clean "testOnly org.broadinstitute.dsde.rawls.consumer.*"
```

Comment:
Do I need to update line 111 too since the rest of rawls is updating its base image?  I got "could not retrieve sbt 1.9.9" when trying this locally but it seems like the GHAs are working.

Reply:
I notice the **publish-contracts** job in `.github/workflows/consumer_contract_tests.yml#L138-L144` is missing `release-tag` input parameter. Here's the suggested change.

```
inputs: '{
            "run-name": "${{ env.PUBLISH_CONTRACTS_RUN_NAME }}",
            "pact-b64": "${{ needs.rawls-consumer-contract-tests.outputs.pact-b64 }}",
            "repo-owner": "${{ github.repository_owner }}",
            "repo-name": "${{ github.event.repository.name }}",
            "repo-branch": "${{ needs.init-github-context.outputs.repo-branch }}",
            "release-tag": "${{ needs.init-github-context.outputs.repo-version }}"
          }'
```

Reply:
Please update 
```
  rawls-consumer-contract-tests:
    runs-on: ubuntu-latest
    needs: [ init-github-context ]
```

to 
```
  rawls-consumer-contract-tests:
    runs-on: ubuntu-latest
    needs: [ bump-check, init-github-context ]
    if: ${{ needs.bump-check.outputs.is-bump == 'no' }}
```

Reply:
Also, please update
```
  publish-contracts:
    runs-on: ubuntu-latest
    if: ${{ needs.init-github-context.outputs.fork == 'false' }}
    needs:
      - init-github-context
      - rawls-consumer-contract-tests
```

to
```
  publish-contracts:
    runs-on: ubuntu-latest
    if: ${{ (needs.init-github-context.outputs.fork == 'false') && (needs.bump-check.outputs.is-bump == 'no') }}
    needs:
      - bump-check
      - regulated-tag-job
      - init-github-context
      - rawls-consumer-contract-tests
```

Reply:
In `can-i-deploy`, update

```
"version": "${{ needs.init-github-context.outputs.repo-version }}"
```
to
```
"version": "${{ needs.regulated-tag-job.outputs.new-tag }}"
```